.gitignore

```diff
@@ -0,0 +1,6 @@
+.terraform/
+/.idea/
```

Comment:
Why is the the prefixed '/' required? Same comment applies for line #4.

Reply:
Have to admit, I copied this .gitignore from the internet based on an example one that showed which terraform files are best to be left out. addressed